eng/docgeneration/Generate-DocIndex.ps1

```diff
@@ -1,8 +1,23 @@
+# Generates an index page for cataloging different versions of the Docs
 [CmdletBinding()]
 Param (
     $RepoRoot,
-    $DocGenDir
+    $DocGenDir,
+    $lang = "java",
```

Comment:
You should be able to use the $Language from https://github.com/Azure/azure-sdk-for-js/blob/master/eng/scripts/Language-Settings.ps1#L1